airflow/models/dagbag.py

```diff
@@ -47,6 +47,16 @@
 from airflow.utils.session import provide_session
 from airflow.utils.timeout import timeout
 
+# TODO: Remove once deprecated
+if hasattr(settings, "policy"):
+    warnings.warn(
+        "Using `policy` in airflow_local_settings.py is deprecated. "
+        "Please rename your `policy` to `task_policy`.",
+        DeprecationWarning,
+        stacklevel=2,
+    )
+    setattr(settings, "task_policy", settings.policy)  # pylint: disable=no-member
```

Comment:
This should be done in `airflow.settings.import_local_settings` function please.

Reply:
That was my intial idea but somehow I decided to do this in dagbag 🤷‍♂️  Done in a250f41

Reply:
I'm just conscious that DagBag may go away at some point soon -- it does a lot less than it used to.

Also top-level code is a bit of a smell to me.